google/cloud/aiplatform/models.py

```diff
@@ -1737,12 +1777,95 @@ def container_spec(self) -> Optional[aiplatform.gapic.ModelContainerSpec]:
         self._assert_gca_resource_is_available()
         return getattr(self._gca_resource, "container_spec")
 
+    @property
+    def version_id(self) -> str:
+        """The version ID of the model.
+        A new version is committed when a new model version is uploaded or
+        trained under an existing model id. It is an auto-incrementing decimal
+        number in string representation."""
+        self._assert_gca_resource_is_available()
+        return getattr(self._gca_resource, "version_id")
+
+    @property
+    def version_aliases(self) -> Sequence[str]:
+        """User provided version aliases so that a model version can be referenced via
+        alias (i.e. projects/{project}/locations/{location}/models/{model_id}@{version_alias}
+        instead of auto-generated version id (i.e.
+        projects/{project}/locations/{location}/models/{model_id}@{version_id}).
+        The format is [a-z][a-zA-Z0-9-]{0,126}[a-z0-9] to distinguish from
+        version_id. A default version alias will be created for the first version
+        of the model, and there must be exactly one default version alias for a model."""
+        self._assert_gca_resource_is_available()
```

Comment:
Do these assertions raise a runtime error for the user? If not, I think it'll be a good idea to do so, along with a clear error message explaining that the model resource isn't available.

Reply:
Yes, its in base.py